filter-processor/src/test/java/org/springframework/cloud/stream/module/filter/FilterProcessorApplicationTests.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.filter;
+
+import static org.junit.Assert.*;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.cloud.stream.annotation.Input;
+import org.springframework.cloud.stream.annotation.ModuleChannels;
+import org.springframework.cloud.stream.annotation.Output;
+import org.springframework.cloud.stream.annotation.Processor;
+import org.springframework.cloud.stream.annotation.Sink;
+import org.springframework.cloud.stream.annotation.Source;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = FilterProcessorApplication.class)
+@WebAppConfiguration
+@DirtiesContext
+public class FilterProcessorApplicationTests {
+
+	@Autowired
+	@ModuleChannels(FilterProcessor.class)
+	private Processor processor;
+
+	@Autowired
+	private Processor same;
+
+	@Output(Source.OUTPUT)
+	private MessageChannel output;
+
+	@Input(Sink.INPUT)
+	private MessageChannel input;
+
+	@Test
+	@Ignore
+	public void contextLoads() {
```

Comment:
first of all, the `@Ignore` should not be there.
This is indeed a copy/paste, but could serve a purpose (the `@Output/@Input` are bogus though): it proves that the module loads with default properties.
Currently working on other tests with non-default props (as discussed in the Slack channel with Stéphane)